app/jobs/reindex_admin_sets_job.rb

```diff
@@ -2,6 +2,8 @@
 
 class ReindexAdminSetsJob < ApplicationJob
   def perform
-    AdminSet.find_each(&:update_index)
+    AdministrativeSet.find_each do |admin_set|
```

Comment:
This needs to be change to use the config collection class. Possibly should also reference multiple models?